### DISCUSSION TITLE
Ứng dụng chat nặc danh mã hoá 2 đầu sử dụng NeoSocket. Người dùng có thể bắt đầu 1 đoạn chat mà không cần đăng nhập hay xác thực mà vẫn đảm bảo tính bảo mật.

### DISCUSSION THREAD

**Discussion by giscus** [thangchiba/ThangChibaBlog#6] [Announcements]
# Ứng dụng chat nặc danh mã hoá 2 đầu sử dụng NeoSocket. Người dùng có thể bắt đầu 1 đoạn chat mà không cần đăng nhập hay xác thực mà vẫn đảm bảo tính bảo mật.

Cypher.

https://blog.thangchiba.com/vi/project/cypher/

**Comment by tyanfarm** [comment]
a cho em xin link git em tham khảo được k ạ

**Reply by thangchiba** [reply]
Cypher mình mới push FE lên thôi. BE chưa có thời gian đóng gói để đẩy lên. B thông cảm xem tạm trước nhé : https://github.com/thangchiba/Cypher
